runtime-light/stdlib/fork/fork-api.cpp

```diff
@@ -11,6 +11,10 @@
 #include "runtime-light/coroutine/awaitable.h"
 #include "runtime-light/coroutine/task.h"
 
+int64_t f$get_running_fork_id() noexcept {
```

Comment:
Let's define it in header, so the compiler can inline it.

P.S.: please don't add `inline` keyword in header